<?xml version="1.0" ?>
<robot name="test">
    
    <group name="all">
        <joint name="object_joint"/>
        
        <joint name="FFJ4"/>
        <joint name="FFJ3"/>
        <joint name="FFJ2"/>
        <joint name="FFJ1"/>
        
        <joint name="MFJ4"/>
        <joint name="MFJ3"/>
        <joint name="MFJ2"/>
        <joint name="MFJ1"/>
        
        <joint name="RFJ4"/>
        <joint name="RFJ3"/>
        <joint name="RFJ2"/>
        <joint name="RFJ1"/>
        
        <joint name="LFJ5"/>
        <joint name="LFJ4"/>
        <joint name="LFJ3"/>
        <joint name="LFJ2"/>
        <joint name="LFJ1"/>
        
        <joint name="THJ5"/>
        <joint name="THJ4"/>
        <joint name="THJ3"/>
        <joint name="THJ2"/>
        <joint name="THJ1"/>
        
        <joint name="WRJ1"/>
        <joint name="WRJ2"/>
        
    </group>
    
    <group name="robot">
        
        <joint name="FFJ4"/>
        <joint name="FFJ3"/>
        <joint name="FFJ2"/>
        <joint name="FFJ1"/>
        
        <joint name="MFJ4"/>
        <joint name="MFJ3"/>
        <joint name="MFJ2"/>
        <joint name="MFJ1"/>
        
        <joint name="RFJ4"/>
        <joint name="RFJ3"/>
        <joint name="RFJ2"/>
        <joint name="RFJ1"/>
        
        <joint name="LFJ5"/>
        <joint name="LFJ4"/>
        <joint name="LFJ3"/>
        <joint name="LFJ2"/>
        <joint name="LFJ1"/>
        
        <joint name="THJ5"/>
        <joint name="THJ4"/>
        <joint name="THJ3"/>
        <joint name="THJ2"/>
        <joint name="THJ1"/>
        
        <joint name="WRJ1"/>
        <joint name="WRJ2"/>
        
    </group>
    
</robot>
